packages/edit-site/src/store/reducer.js

```diff
@@ -169,24 +158,32 @@ export function page( state = {}, action ) {
 }
 
 /**
- * Reducer returning the site's `show_on_front` setting.
+ * Reducer for information about the site's homepage.
  *
  * @param {Object} state Current state.
+ * @param {Object} action Dispatched action.
  *
  * @return {Object} Updated state.
  */
-export function showOnFront( state ) {
+export function home( state = {}, action ) {
```

Comment:
It seems like reducer `state` would always be defined here. Is there an instance where we need the default param?

Reply:
I think you're right, but I like to add it in case we need to "home" from the initial state passed into registerStore